CIP-????/README.md

```diff
@@ -0,0 +1,156 @@
+---
+CIP: ?
+Title: Better script purposes
```

Comment:
If there is one dissenting comment about these script purposes not being unilaterally "better" I would propose one of the following:
* Updated Script Purposes (if reviewers are indifferent)
* Optimised Script Purposes (if reviewers are enthusiastic... and if this title is in fact accurate)

At this point that should be measurable by those already tagged (@lehins @WhatisRT @MicroProofs @fallen-icarus).

Reply:
thanks @zliu41 - will highlight for discussion the title issue & your comment and brainstorm a "better" title at the next CIP meeting (https://hackmd.io/@cip-editors/117) where this is up for `Triage`.  @michele-nuzzi you are, as always, welcome to come & introduce the proposal and discuss this & other introductory questions.

Reply:
Resolved by author in https://github.com/cardano-foundation/CIPs/pull/1072#event-19159347835.